content/admin/enterprise-management/initiating-a-failover-to-your-replica-appliance.md

```diff
@@ -30,6 +30,15 @@ The time required to failover depends on how long it takes to manually promote t
 5. Update the DNS record to point to the IP address of the replica. Traffic is directed to the replica after the TTL period elapses. If you are using a load balancer, ensure it is configured to send traffic to the replica.
 6. Notify users that they can resume normal operations.
 7. If desired, set up replication from the new primary to existing appliances and the previous primary. For more information, see "[About high availability configuration](/enterprise/{{ currentVersion }}/admin/guides/installation/about-high-availability-configuration/#utilities-for-replication-management)."
+8. Appliances you do not intend to setup replication to that were part of the high availability configuration prior the failover, need to be removed from the high availability configuration by UUID.
+    - On the former appliances, get their UUID via `cat /data/user/common/uuid`.
+      ```shell
+      $ cat /data/user/common/uuid
+      ```
+    - On the new primary, remove the UUIDs using `ghe-repl-teardown`. Please replace *`UUID`* with a UUID you retrieved in the previous step.
+      ```shell
+      $ ghe-repl-teardown -u <em>UUNID</em>
```

Comment:
`s/UUNID/UUID/` small typo

Reply:
Hmm not sure why this only just popped up in my notifications given it was merged a month ago. I'll open a new PR to fix the typo.